internal/api/handlers_test.go

```diff
@@ -379,43 +384,43 @@ func TestGoCertUsersHandlers(t *testing.T) {
 
 	client := ts.Client()
 
+	var adminToken string
+	var nonAdminToken string
+	t.Run("prepare user accounts and tokens", prepareUserAccounts(ts.URL, client, &adminToken, &nonAdminToken))
+
 	testCases := []struct {
 		desc     string
 		method   string
 		path     string
 		data     string
+		auth     string
 		response string
 		status   int
 	}{
-		{
-			desc:     "Create first user success",
-			method:   "POST",
-			path:     "/api/v1/accounts",
-			data:     adminUser,
-			response: "{\"id\":1}",
-			status:   http.StatusCreated,
-		},
```

Comment:
Why are we removing this test case?

Reply:
it is included in the previous test case that creates 2 accounts, logs in as them and saves their login tokens for the rest of the test suite